.babelrc

```diff
@@ -1,14 +1,15 @@
 {
   "presets": [
-    ["env", {
+    ["@babel/env", {
       "loose": true
     }],
-    "stage-1",
-    "react"
+    "@babel/react"
   ],
   "plugins": [
-    "dev-expression",
-    "transform-runtime", // Need stage-3 Symbol polyfill.
-    "add-module-exports"
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-export-default-from",
+    "@babel/plugin-proposal-export-namespace-from",
+    "@babel/transform-runtime",
```

Comment:
Okay, so this raises a question – by default, runtime in Babel 7 no longer supplies polyfills.

So... under what conditions will this still work right? Do we need to be like react-relay-network-modern and ask users to pull in regenerator themselves?

Reply:
wouldn't we still need some sort of ES7-ish symbol polyfill then?

i didn't do it by hand because i was afraid of screwing up something around try-catch. the resolver code is much easier to write in this generator-ish style

Reply:
> wouldn't we still need some sort of ES7-ish symbol polyfill then?

technically, but I'm pretty sure you don't actually need a polyfill, everyone seems to just do what iterall does: https://github.com/leebyron/iterall/blob/master/index.mjs#L466 Aollo subscriptions is doing it anyway.

> the resolver code is much easier to write in this generator-ish style

Yeah totally, it's be a tradeoff for small deps i guess?

Reply:
> Do we need to be like react-relay-network-modern and ask users to pull in regenerator themselves?

I would suggest that this be avoided as it's _real_ annoying IMO. Better to just bundle it with a plugin and ship it out.